workers/default/ocr/tiff/tiff_to_text.py

```diff
@@ -0,0 +1,39 @@
+import logging
+
+from PIL import Image
+import pyocr
+
+log = logging.getLogger(__name__)
+
+
+def tiff_to_text(image_path, language='eng'):
```

Comment:
Wir sollten diskutieren, ob wir die Sprache als Parameter im Frontend übergeben oder ob wir einen Task zu Language-Detection implementieren.

Reply:
Außerdem hatten wir uns darauf geeinigt, Formate in drei Buchstaben anzugeben. Die Funktion müsste also `tif_to_txt` heißen.